### DISCUSSION TITLE
node-fetch run nginx cpu 100%  Downtime

### DISCUSSION THREAD

**Discussion by JamieDIng** [nuxt/nuxt#17010] [Questions]
Web Server configuration
[2-core 2G memory 4m] (nodev16.17.0 + pm2) 
There is nothing wrong with the website itself
But because my interface is on another server
When I visit the website of nuxt, I will increase the CPU of the interface server to 100%
The API nginx server process finds node fetch requests 3 times per second
![image](https://user-images.githubusercontent.com/5538427/186631658-2a52e04b-b20b-4586-b670-1c33603bc4da.png)
I need help
How can I troubleshoot the problem
I can't find the problem~~~
Or are there too many request methods in the code

**Comment by TheAlexLichter** [comment]
Well, without any code or minimal reproduction it is really hard to pinpoint the issue ☹️

**Reply by JamieDIng** [reply]
> Well, without any code or minimal reproduction it is really hard to pinpoint the issue ☹️
OK, I'll try to upload all the files to GitH

**Reply by JamieDIng** [reply]
I found that I used refresh () in many places ( 
For example: 
pages as pagination
Go back to the home page and get the data on the first page )
![image](https://user-images.githubusercontent.com/5538427/186675963-ce96aaeb-fefc-486b-b2bb-8cf3a647c535.png)
Then my API uses WordPress + graphiql
Maybe the performance of WP is not good~~~
Therefore, when I press F5 crazily, the CPU will peak
